scripts/docker/tasking-manager/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN pip install --no-cache-dir -r requirements.txt
 ## CLIENT CONFIGURATION
 
 # Install nodejs.
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - \
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
```

Comment:
Buster has nodejs in its package list. We may not need this if we do `apt install nodejs` 

https://packages.debian.org/buster/nodejs

Reply:
Oh, right. Very good. Removed this entirely on the buster images. Thanks.